source/connect/connect-from-the-command-line.txt

```diff
@@ -38,23 +38,47 @@ Command Line Connection Specification
 -------------------------------------
 
 The command line invocation for |compass-short| has two components, the 
-path to the |compass-short| executable and a connection string. The
-format is:
+path to the |compass-short| executable and a connection string, you can
```

Comment:
[suggestion] This is a run-on sentence.

Reply:
Updated, broke this into 2 separate sentences.